MIAUI/ViewModels/TasksViewModel.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
+using Google.Apis.Util.Store;
+using Microsoft.Maui.Controls;
+
+using MIAUI.Model;
+using MIAUI.Services;
+//using static Android.Content.ClipData;
+using Google.Apis;
+using CommunityToolkit.Mvvm.ComponentModel;
+using System.Collections.ObjectModel;
+using CommunityToolkit.Mvvm.Input;
+using System.Threading.Tasks;
+using MIAUI.Views;
+using Task = System.Threading.Tasks.Task;
+
+namespace MIAUI.ViewModels
+{
+    public partial class TasksViewModel : BaseViewModel
+    {
+        public TasksViewModel()
+        {
+            Items = new ObservableCollection<string>();
+        }
+
+        [ObservableProperty]
+        ObservableCollection<string> items;
+
+        [ObservableProperty]
+        bool isBusy = false;
+
+        [ObservableProperty]
+        string taskName; //Task's title
+
+        [ICommand]
```

Comment:
I happen to notice in the latest pre-release of the CommunityToolkit.Mvvm that ICommand is gone and everything seems to be RelayCommand. Probably no reason to adopt a pre-release, but just a heads up.